Client/Frontend/Browser/TabCell.swift

```diff
@@ -0,0 +1,269 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+
+class TabCell: UICollectionViewCell, TabTrayCell {
+
+    enum Style {
+        case light
+        case dark
+    }
+
+    static let reuseIdentifier = "TabCellIdentifier"
```

Comment:
We can use the `ReusableCell` protocol for this, and never have to worry about it again. :)